docs/conf/atlantic/2023/writing-day.rst

```diff
@@ -12,15 +12,58 @@ Schedule
 - Date & Time: **{{date.day_two.dotw}}, {{date.day_two.date}}, {{date.day_two.writing_day_time}} {{tz}}**.
 - Location: **{{about.venue}}**.
 
-Project list
-------------
+Writing Day projects are welcome to join us for the entire event, the first session, or the second session. 
```

Comment:
Or both?

Reply:
It says that by "for the entire event", right?

Reply:
lol, sure does :-D